backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/aries/ProofManager.java

```diff
@@ -253,6 +252,12 @@ public void sendProofProposal(@NonNull UUID partnerId, @NonNull UUID myCredentia
         return result;
     }
 
+    public List<AriesProofExchange> getAllPartnerProof() {
```

Comment:
Filter by partner id?

Reply:
That method could also exist, but it doesn't have to be filtered. Another issue is that the proof-exchange.id is the only variable on the presentationdetailspage, so we could be very strict and add the partner-id to that route, but it's not necessary. 

let me know if you have a specific endpoint design you have, but i think all of this is valid as well.